pyproject.toml

```diff
@@ -70,6 +70,7 @@ pytest-cov = "^4.0"
 pytest-mock = "^3.9"
 pytest-randomly = "^3.12"
 pytest-xdist = { version = "^3.1", extras = ["psutil"] }
+pytest-httpserver = "^1.0.8"
```

Comment:
Do we need the extra dependency? we already have `httpretty`, isn't that enough?